### DISCUSSION TITLE
What's your vision on openmage-future?

### DISCUSSION THREAD

**Discussion by kiatng** [openmage-strict/openmage-future#1] [General]
Where are you taking this?

**Comment by sreichel** [comment]
Sorry for late response ... was not active past weeks.

My (unsorted) goals are ...

- to improve code-quality
  - cover template files with phpstan/phpcs checks (started)
  - add phpmd (started)
  - Sonar cloud fixes  ...  (very long term)
- modulize Mage components (long term)
- finish rector to update 3rd party -code (long term)
  - use strict types everywhere
-  maybe switch to zf1s
- re-add my closed PRs
- writing unit tests
